tools/validate/src/dependencies/validateListedDependencies.ts

```diff
@@ -38,16 +38,19 @@ export function validateListedDependencies(
     const listedButOnlyUsedAsDev = listedDependencies.filter(
       listedDepName =>
         !importedPackagesInSourceFile.includes(listedDepName) &&
-        !ignoreDependencies.includes(listedDepName),
+        !externalDependencies.some(glob => {
+          const regexPattern = glob.replace('*', '.+'); // a super rough conversion of glob pattern to regexp
```

Comment:
This is not blocking or anything but can you give an example of whats happening here?

Reply:
👍 Moved the logic to a utility function. We're converting a [`glob` ](https://en.wikipedia.org/wiki/Glob_(programming))pattern to RegEx 

`globToRegex('@leafygreen-ui/*') // /@leafygreen-ui\/.+/`